From 0fa8ea7bf78c193ad42b61fabb6edd97cbc07674 Mon Sep 17 00:00:00 2001
From: Ting-Ying Li <tingying.li@cypress.com>
Date: Mon, 9 Aug 2021 01:29:12 -0500
Subject: [PATCH 091/179] brcmfmac: Fix structure size for WPA3 external SAE

Add pmkid parameter in "brcmf_auth_req_status_le" structure to
align the buffer size defined in firmware "wl_auth_req_status"
structure.


Signed-off-by: Ting-Ying Li <tingying.li@infineon.com>
---
 drivers/net/wireless/broadcom/brcm80211/brcmfmac/fwil_types.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/net/wireless/broadcom/brcm80211/brcmfmac/fwil_types.h b/drivers/net/wireless/broadcom/brcm80211/brcmfmac/fwil_types.h
index afedb7627029..aa4450d3c824 100644
--- a/drivers/net/wireless/broadcom/brcm80211/brcmfmac/fwil_types.h
+++ b/drivers/net/wireless/broadcom/brcm80211/brcmfmac/fwil_types.h
@@ -562,6 +562,7 @@ struct brcmf_auth_req_status_le {
 	u8 peer_mac[ETH_ALEN];
 	__le32 ssid_len;
 	u8 ssid[IEEE80211_MAX_SSID_LEN];
+	u8 pmkid[WLAN_PMKID_LEN];
 };
 
 /**
-- 
2.17.1

